gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageWriteChannel.java

```diff
@@ -72,6 +73,7 @@ public GoogleCloudStorageWriteChannel(
     this.objectName = objectName;
     this.writeConditions = writeConditions;
     this.metadata = objectMetadata;
+    this.kmsKeyName = null;
```

Comment:
You can not modify existing constructor in new subminor GCS connector version (we don't plan to release new minor version of GCS connector yet) because this will break existing clients, including older Beam SDK versions.

You need to create a new constructor with `kmsKeyName` parameter and pass null to it from this constructor.